lib/Hash/MoreUtils.pm

```diff
@@ -57,11 +57,18 @@ If no C<LIST> is given, all keys are assumed as C<LIST>.
 
 =head2 C<slice_missing> HASHREF[, LIST]
 
-Returns a hash containing the (key => undef) pair for every
-LIST (as key) item not in hashref.
+Returns a C<HASHREF> containing the (key => undef) pair for every
+C<LIST> (as key) item not in hashref.
 
 If no C<LIST> is given, the returned map is empty.
 
+=head2 C<slice_notdef> HASHREF[, LIST]
+
+As notdef understood as undef this function returns a
```

Comment:
In case you're inconvenient with slice_notdef and prefer slice_undef, feel free to rename the function to make yourself more convenient.